src/linter.h

```diff
@@ -18,39 +18,77 @@
 #define SRC_LINTER_H_
 
 #include <vector>
+#include <cstdio>
+#include <string>
 
 #include "absl/strings/string_view.h"
 #include "zetasql/base/status.h"
 #include "zetasql/parser/parse_tree_visitor.h"
+#include "zetasql/parser/parse_tree.h"
 #include "zetasql/parser/parser.h"
 #include "zetasql/public/parse_helpers.h"
+#include "zetasql/public/parse_resume_location.h"
 
 namespace zetasql {
 
 namespace linter {
 
-class KeywordExtractor {
+// It gets rule and applies that rule to every ASTnode it visit
+class RuleVisitor : public zetasql::NonRecursiveParseTreeVisitor {
```

Comment:
you don't have to specify zetasql:: namespace since you're already inside it. (This applies to all places where you use zetasql:: apis)

Reply:
Erased all zetasql:: prefixes :)